cypress/integration/ete/sign_in.spec.js

```diff
@@ -7,20 +7,32 @@ const existing = {
   password: 'existing_password',
 };
 describe('Sign in flow', () => {
-  it('links to the correct places', () => {
+  const returnUrl =
+    'https://www.theguardian.com/world/2013/jun/09/edward-snowden-nsa-whistleblower-surveillance';
+
+  it('links to the Google terms of service page', () => {
```

Comment:
Update `Terms` tests for sign in too as `Terms` component was modified as part of this work.